drivers/power/bd7181x-power.c

```diff
@@ -220,31 +220,31 @@ static const struct tn_power_values_st TN_POWER_AVENTURA = {
 static const struct tn_power_values_st TN_POWER_TERRA = {
 	.term_current = 0x05,
 	.capacity = 2300,
-	.low_voltage_th = 0x00BF, // 0x00BF (191) * 16mV = 3.056V,
+	.low_voltage_th = 0x00D6, // 0x00D6 (214) * 16mV = 3.4297V,
```

Comment:
como puede ser este valor más alto que el penúltimo valor de la tabla?

Reply:
You are right, SIGTERM will come before arriving 0%.